framework/src/org/checkerframework/framework/type/GenericAnnotatedTypeFactory.java

```diff
@@ -1323,6 +1357,29 @@ protected void checkAndPerformFlowAnalysis(Tree tree) {
     }
 
     /**
+     * Returns the inferred value (by the org.checkerframework.dataflow analysis) for a given node.
+     */
+    private Value getInferredValueFor(Node node) {
```

Comment:
The method is private and I don't see a caller in this class. What is it used for?